c++ training/SteamProject/Videogame/Steam.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "CategoryContainer.h"
+
+
+
+class FSteam {
+private:
+	FCategoryContainer CategoryContainer;
+	FCategory Uncategorized = FCategory("Uncategorized");
+
+	int NumberOfGames;
+
+	enum class MenuCommand
+	{
+		MainMenu = 1,
+		AddGamesMenu,
+		CategoryManagerMenu,
+		GameDisplayerMenu,
+		Exit
+	};
+	MenuCommand ActiveCommand;
+	
+	void OpenMainMenu();
+	void OpenAddGamesMenu();
+	void OpenCategoryManagerMenu();
+	void OpenGameDisplayerMenu();
+
+	bool IsInteger(const float Float) const;
+	bool IsInRange(const float Value, const int LowerBound, const int UpperBound) const;
+	int ValidateInputAndReturnValidatedInput(float FirstInput, const std::string Message, const int LowerBound, const int UpperBound);
```

Comment:
about this name... we have to be the most descriptive possible but is it necessary that its function have a name so large?

Reply:
I mean, I also hesitated when I named it, the ValidateInput name could tell me it's trying to validate the input, but it isn't quite clear I want a validated input to be returned. That could, however, be dealt with pointers, though we cannot really use them yet, so I'm a bit confused here on how to proceed.